Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -40,6 +41,17 @@ VolumeGC::VolumeGC(std::unique_ptr<BlobReader> reader) : m_reader(std::move(read
   };
 
   m_converted_banner = [this] { return LoadBannerFile(); };
+
+  constexpr u32 BTID_MAGIC = 0x44495442;
+  BootID triforce_header;
+  auto tmp_fs = std::make_unique<FileSystemGCWii>(this, PARTITION_NONE);
+  if (tmp_fs)
+  {
+    const u64 file_size = ReadFile(*this, PARTITION_NONE, tmp_fs->FindFileInfo("boot.id").get(),
```

Comment:
Missing `nullptr` check for `FindFileInfo` return value.

Reply:
Not really sure how to implement that, would something like this work?

    
    if (FindFileInfo("boot.id").get() != nullptr && file_size >= 4 && triforce_magic == BTID_MAGIC)
      m_is_triforce = true;`

Reply:
Store the result of FindFileInfo in a variable, then check the variable against nullptr, then pass the variable to ReadFile.